services/base-jdbc/src/main/java/org/eclipse/hono/service/base/jdbc/store/device/TableManagementStore.java

```diff
@@ -208,8 +278,8 @@ public TableManagementStore(final JDBCClient client, final Tracer tracer, final
      * It returns the plain result set from the query, which may also be empty.
      *
      * @param connection The connection to use.
-     * @param key The key of the device.
-     * @param span The span to contribute to.
+     * @param key        The key of the device.
+     * @param span       The span to contribute to.
```

Comment:
can you please remove the tabs here and in the other locations?

Reply:
Done